src/ac-power/meson.build

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+executable(
+        'systemd-ac-power',
+        'ac-power.c',
+        include_directories : includes,
+        link_with : [libshared],
+        install_rpath : rootlibexecdir,
+        install : true,
+        install_dir : rootlibexecdir)
```

Comment:
Sorry, no. We need executables to be in the main build directory. In fact, we should move (some?) `.so` libraries there too, to make it easier to set `LD_LIBRARY_PATH`.

Reply:
Any specific reason for that? Is it to make it easier to run in-place?

Reply:
That's the major reason. Secondary reason is to be able to set one path in PATH or LD_LIBRARY_PATH, without having to hunt down the full list and then keep it up-to-date in external things that want to build systemd and run stuff from the build directory.

Reply:
I see. I must say I quite liked having everything a bit more tidied up, but what you say makes sense.

Reply:
Hmm, so, the main commit of this PR will not be accepted... That's sad news for me. But ok. I will drop it.

Reply:
Yeah ;( I think we can treat this as a shortcoming of meson. It would be nice if meson would be more flexible in separating the definition and the output location of a build step. But it'd not be a trivial thing to do... I know they are already swamped with PRs and issues, so I think we should just accept that this is not going to change any time soon.